README.md

```diff
@@ -65,12 +65,20 @@ To build the project:
 - Add dependency on Scala compiler plugin [Macro Paradise](https://github.com/scalamacros/paradise).
   - This can be done by adding `addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)` to the build settings in build.sbt.
   - An example project, which uses Macro Paradise can be found [here](https://github.com/scalamacros/sbt-example-paradise).
-- Add dependency on this repository.
-  - This can be done by adding the following line to the build.sbt file
-  ```scala
-  lazy val hiddenArgs = ProjectRef(uri("https://github.com/keddelzz/hidden-args.git"), "hiddenargs")
+- Add the following resolver to your `build.sbt`:
+```
+resolvers += Resolver.bintrayRepo("keddelzz", "maven")
+```
+- Add the following dependency to your `build.sbt`:
+
+  - For Scala 2.11:
+  ```
+  libraryDependencies += "com.github.keddelzz.hidden-args" % "hidden-args_2.11" % "0.0.1"
```

Comment:
Hey! Isn't it better to replace `_2.xx` with `%%`? E.g. `libraryDependencies += "com.github.keddelzz.hidden-args" %% "hidden-args" %% "0.0.1"`? It should work with both Scala versions.